app/src/main/java/com/keylesspalace/tusky/TabPreferenceActivity.kt

```diff
@@ -273,7 +274,13 @@ class TabPreferenceActivity : BaseActivity(), Injectable, ItemInteractionListene
     }
 
     private fun showSelectListDialog() {
-        val adapter = ArrayAdapter<MastoList>(this, android.R.layout.simple_list_item_1)
+        val adapter = object : ArrayAdapter<MastoList>(this, android.R.layout.simple_list_item_1) {
+            override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
+                val view = super.getView(position, convertView, parent)
+                getItem(position)?.let { item -> (view as TextView).text = item.title }
```

Comment:
(If I remeber correctly there was a sort of "?as" )

Reply:
`as?` would silently fail. So if the layout was changed but this wasn't we'd not know unless someone told us.  Using `as` like this means it'll crash, which will show up in the logs.